File: R/ex6.R
#!/bin/Rscript

## Exercício 2.11
main <- function()
{
  cat("Entre com o número: ")
  n <- readLines("stdin", n = 1)

  string <- strsplit(as.character(n),"")[[1]]
  
  string <- string[length(string):1]
  
  result <- paste(as.character(n), "->", paste(string, collapse = ""))
  
  return(result)
}

main()

File: py/ex6.py
#!/bin/python

## Exercício 2.11
def main():
  n = str(input("Entre com o número: "))

  rev_number(n)

def rev_number(n):
  string = list(n)

  string.reverse()

  result = n+" -> "+"".join(string)

  print(result)


if __name__ == "__main__":
  main()
